cmd/kube-apiserver/app/server.go

```diff
@@ -593,10 +589,28 @@ func Complete(s *options.ServerRunOptions) (completedServerRunOptions, error) {
 			}
 		}
 
-		s.ServiceAccountIssuer, err = serviceaccount.JWTTokenGenerator(s.Authentication.ServiceAccounts.Issuer, sk)
+		info, err := os.Stat(s.ServiceAccountSigningKeyFile)
```

Comment:
Maybe toggle on the URL. E.g. `unix://var/run/key.sock` is an external signing service and the scheme defaults to file://

Reply:
Added a new flag instead